src/index.html

```diff
@@ -12,6 +12,148 @@
   </head>
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="personal-info">
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input
+            type="text"
+            autocomplete="disable"
+            name="surname"
+            id="surname">
```

Comment:
```suggestion
          <input
            type="text"
            autocomplete="disable"
            name="surname"
            id="surname"
          >
```
Закриваюча дужка має бути на новому рядку і на тому ж рівні відступів, що і відкриваюча